### ISSUE TITLE
deleting notes?

### ISSUE THREAD

**Issue by tkossak** [neilagabriel/vim-geeknote#23]
Is there a way to delete notes? When i delete the line in the navigation buffer and save it, the line just comes back.

**Comment by neilagabriel** [comment]
No, but oddly enough I was in the process adding support for this today.
Stay tuned, it will be possible soon.

Unfortunately, I won't be able to support deleting notebooks, just notes.
The Evernote API doesn't allow 3rd party apps from deleting notebooks (I
don't know why). The best a 3rd party app can do is "delete" a note by
essentially moving it to the trash (it cannot delete it permanently).

**Comment by igorcavalcante** [comment]
+1 :)

**Comment by jvican** [comment]
It would be awesome to have this feature.

**Comment by hobbestigrou** [comment]
Effectively it will be great to have this feature.
